PointerParameters.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#define Size 10
+
+int sum(int *start, int *end)
+{
+   int initialization = 0;
```

Comment:
I think this needs a better name. This is used to store the running total, but it's name is `initialization`.